### ISSUE TITLE
Error: Call to function "merge" failed using cloudfront_origins

### ISSUE THREAD

**Issue by jagarzonhuge** [dealmore/terraform-aws-next-js#66]
Using the input `cloudfront_origins` results in a terraform error:
```sh
Error: Error in function call

  on .terraform/modules/tf_next/main.tf line 173, in locals:
 173:   proxy_cloudfront_origins = var.cloudfront_origins != null ? merge(
 174:     local.next_image_cloudfront_origins,
 175:     var.cloudfront_origins
 176:   ) : local.next_image_cloudfront_origins
    |----------------
    | local.next_image_cloudfront_origins is list of object with 1 element
    | var.cloudfront_origins is list of object with 1 element

Call to function "merge" failed: arguments must be maps or objects, got "list
of object".
```

**Comment by ofhouse** [comment]
Are you able to share what you passed as argument?
It's not very well documented (since we will provide a better solution for it soon #55) but it requires a very specific format in order to make it work.

**Comment by jagarzonhuge** [comment]
I tried with an empty list `[]` and with this:
```terraform
cloudfront_origins           = [{
    domain_name: aws_s3_bucket.storybook.bucket_regional_domain_name,
    origin_id: local.s3_origin_id,

    s3_origin_config: {
      origin_access_identity: "origin-access-identity/cloudfront/XXXXXXXXXXXXXX"
    }
}]
```
As the error message says, the module is trying to merge 2 `lists of objects` but the `merge` function only works with `maps/objects`

**Comment by ofhouse** [comment]
@jagarzonhuge Were you able to solve the issue or does the problem persist?

**Comment by ofhouse** [comment]
This has been fixed in [`v0.7.4`](https://github.com/dealmore/terraform-aws-next-js/releases/tag/v0.7.4).
